spec/integration/.kitchen.yml

```diff
@@ -0,0 +1,19 @@
+---
+driver:
+  name: docker
+  image: centos:6.7
+  use_sudo: true
```

Comment:
Does travis require sudo for the docker binary, or just here? https://github.com/neillturner/kitchen-puppet/pull/96/files#diff-354f30a63fb0907d4ad57269548329e3R7

I'd like to be able to have users run tests locally as well as with CI.

We can set `KITCHEN_LOCAL_YAML` and just eat the cost of maintaining two separate kitchen config files. Such that the travis exec, runs a `.kitchen.travis.yml`, as set by the environment variable.